_posts/2019-02-11-bootstrap-4-3-0.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: Bootstrap 4.3.0
+video: htgr3pvBr-I
+---
+
+Bootstrap v4.3 has landed with over 120 combined closed issues and merged pull requests. This release brings improvements to our utilities, some prep work for moving on to v5's development, and the standard bug fixes and documentation updates.
+
+During our last release, we shared a small preview of where we're taking the project next. That's getting clearer in the coming weeks as our attention turns towards embracing Hugo for ultra fast docs development, removing jQuery in favor of regular JavaScript, and addressing our growing code base.
+
+Keep reading for v4.3 highlights, and see you soon with more details on v5!
+
+## Highlights
+
+We've added some new utilities and deprecated some unused code. Here are the key changes in v4.3, broken down by new, improved, fixed, and deprecated.
+
+- **New:** Added `.stretched-link` utility to make any anchor the size of it's nearest `position: relative` parent, perfect for entirely clickable cards!
+- **New:** Added `.text-break` utility for applying `word-break: break-word`
+- **New:** Added `.rounded-sm` and `.rounded-lg` for small and large `border-radius`.
+- **New:** Added `.modal-dialog-scrollable` modifier class for scrolling content _within_ a modal.
+- **New:** Added responsive `.list-group-horizontal` modifier classes for displaying list groups as a horizontal row.
+- **Improved:** Reduced our compiled CSS by using `null` for variables that by default inherit their values from other elements (e.g., `$headings-color` was `inherit` and is now `null` until you modifier it in your custom CSS).
+- **Improved:** Simplified popover arrow CSS by generating arrows on the `.popover` instead of an additional element, `.popover-arrow`.
```

Comment:
Couldn't find anything related to this, do you have a link of the PR, @mdo.

Reply:
I also can't find anything about this in the generated css https://github.com/twbs/bootstrap/pull/28228/files

Reply:
Ah yes, I misread the PR notes for that one. Removing now.